code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -52,20 +52,44 @@
 	new foldable(get_turf(src))
 	qdel(src)
 
+/obj/item/weapon/storage/box/survival
+	name = "crew survival kit"
+	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	icon_state = "survival"
 /obj/item/weapon/storage/box/survival/New()
 	..()
 	new /obj/item/clothing/mask/breath(src)
 	new /obj/item/weapon/tank/emergency/oxygen(src)
-
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector(src)
+	new /obj/item/stack/medical/bruise_pack(src)
+	new /obj/item/device/flashlight/glowstick(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/candy/proteinbar(src)
+
+/obj/item/weapon/storage/box/vox
+	name = "vox survival kit"
+	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and black stripe indicate this one contains nitrogen."

```

Comment:
Should be plasma, not nitrogen. Unless you want all the vox to suffocate, which I support.

Reply:
Fixed.